packages/ui/src/components/Popover/__stories__/Template.stories.tsx

```diff
@@ -7,7 +7,7 @@ export const Template: StoryFn<typeof Popover> = props => (
     visible
     {...props}
     title="Popover Title"
-    content="This is a simple text content inside the popover. You can customize it by passing text into content property."
```

Comment:
optional mb previous story was a more recommended usage especially for template story ? We could go back to previous story and then decide if a react node as content could have a dedicated story

Reply:
@fabienhebert my bad I forgot to remove the test